projects/aca-shared/src/lib/components/open-in-app/open-in-app.component.scss

```diff
@@ -15,3 +15,7 @@
 .open-in-app.mat-button {
   overflow-x: hidden;
 }
+
+.open-in-app.cdk-program-focused .mat-button-focus-overlay {
```

Comment:
Can you add space after first class, pls `.open-in-app .cdk-program-focused .mat-button-focus-overlay`

Reply:
yess sure, i added a gap between the classes.

Reply:
@jatin2008 @nikita-web-ua  is that still working when space is added? Space makes huge difference for styles because selector without space means that .cdk-program-focused is applied to the same tag like .open-in-app but when you add space it means that .cdk-program-focused is a child of .open-in-app

Reply:
yess I now check it is not working with space added, so i have to revert back this @nikita-web-ua